File: patch_navbar.py
with open('src/components/Navbar.tsx', 'r') as f:
    content = f.read()

desktop_evidence_tab = """
          <button
            onClick={() => { onSelectTab('evidence'); sound.click(); }}
            className={`px-3 py-1.5 rounded-lg text-xs font-mono font-bold flex items-center gap-1.5 transition-colors ${
              currentTab === 'evidence'
                ? 'bg-cyan-500/15 text-cyan-300 border border-cyan-500/40 shadow-sm'
                : 'text-gray-400 hover:text-white hover:bg-gray-800/40'
            }`}
          >
            <Database className="w-3.5 h-3.5" />
            <span>Evidence</span>
          </button>
"""

if "onSelectTab('evidence')" not in content:
    content = content.replace("<span>Rabbit Hole Graph (Map)</span>\n          </button>", "<span>Rabbit Hole Graph (Map)</span>\n          </button>" + desktop_evidence_tab)

mobile_evidence_tab = """
        <button
          onClick={() => { onSelectTab('evidence'); sound.click(); }}
          className={`flex-1 py-1 px-1 rounded-lg text-[11px] font-mono font-bold flex flex-col items-center gap-0.5 transition-colors ${
            currentTab === 'evidence'
              ? 'text-cyan-300 bg-cyan-950/40'
              : 'text-gray-400 hover:text-white'
          }`}
        >
          <Database className="w-4 h-4" />
          <span>Evidence</span>
        </button>
"""

if "<Database className=\"w-4 h-4\" />" not in content:
    content = content.replace("<span>Graph (Map)</span>\n        </button>", "<span>Graph (Map)</span>\n        </button>" + mobile_evidence_tab)
    
if "Database," not in content:
    content = content.replace("FolderArchive,", "FolderArchive, Database,")
elif "Database" not in content:
    content = content.replace("FolderArchive", "FolderArchive, Database")

with open('src/components/Navbar.tsx', 'w') as f:
    f.write(content)

File: patch-navbar.mjs
import fs from 'fs';
let content = fs.readFileSync('src/components/Navbar.tsx', 'utf8');
content = content.replace(
  "currentTab: 'cases' | 'graph' | 'discussions' | 'supporters' | 'evidence';",
  "currentTab: 'cases' | 'graph' | 'discussions' | 'supporters' | 'evidence' | 'workspaces';"
);
content = content.replace(
  "currentTab: 'cases' | 'graph' | 'discussions' | 'supporters';",
  "currentTab: 'cases' | 'graph' | 'discussions' | 'supporters' | 'evidence' | 'workspaces';"
);
content = content.replace(
  "onSelectTab: (tab: 'cases' | 'graph' | 'discussions' | 'supporters' | 'evidence') => void;",
  "onSelectTab: (tab: 'cases' | 'graph' | 'discussions' | 'supporters' | 'evidence' | 'workspaces') => void;"
);
content = content.replace(
  "onSelectTab: (tab: 'cases' | 'graph' | 'discussions' | 'supporters') => void;",
  "onSelectTab: (tab: 'cases' | 'graph' | 'discussions' | 'supporters' | 'evidence' | 'workspaces') => void;"
);

// Add the workspaces button before graph
content = content.replace(
  `<button
            onClick={() => {
              sound.click();
              onSelectTab('graph');`,
  `<button
            onClick={() => {
              sound.click();
              onSelectTab('workspaces');
            }}
            className={\`group flex items-center px-4 py-2 sm:py-3 transition-all duration-300 relative border-r border-white/5 \${
              currentTab === 'workspaces'
                ? 'text-[#00E5FF] bg-white/[0.02]'
                : 'text-gray-400 hover:text-gray-200 hover:bg-white/[0.02]'
            }\`}
          >
            <div className="flex items-center gap-2 relative z-10">
              <FolderArchive className={\`w-4 h-4 transition-transform duration-300 \${currentTab === 'workspaces' ? 'scale-110' : 'group-hover:scale-110'}\`} />
              <span className="text-xs tracking-[0.2em] uppercase font-medium hidden sm:inline-block">Workspace</span>
            </div>
            {currentTab === 'workspaces' && (
              <div className="absolute bottom-0 left-0 right-0 h-[2px] bg-[#00E5FF] shadow-[0_0_10px_rgba(0,229,255,0.5)]" />
            )}
          </button>
          <button
            onClick={() => {
              sound.click();
              onSelectTab('graph');`
);
fs.writeFileSync('src/components/Navbar.tsx', content);
